src/edit.go

```diff
@@ -116,49 +141,74 @@ func EditFile(path string) error {
 	return nil
 }
 
-// Parses k/v pairs and comments from a secret file
+// Validates and parses key/value pairs and comments from the temporary secret file
+//
+// Lines starting with "#" will be recognized as comments. Following lines that also start with "#"
+// will be appended to the first.
+//
+// Lines not starting with "#" will be recognized as secrets. If the key identifier of a secret
+// multiple times the user will get a chance to reedit the secret
+//
 func ParseSecret(path string) (map[string]interface{}, error) {
 
+	var data = make(map[string]interface{})
+	var comment string
+
 	file, err := os.Open(path)
 	if err != nil {
 		return nil, err
 	}
 
 	scanner := bufio.NewScanner(file)
-
-	var data = make(map[string]interface{})
-	var comment string
-
 	for scanner.Scan() {
 
 		line := scanner.Text()
 
 		if line != "" {
+
 			if strings.HasPrefix(line, "#") {
-				// If line is a comment, store it's value for later composition into it's
-				// own k/v pair
+
 				if comment != "" {
-					// If a comment is alreay set, then assume that the comment spans
-					// across multiple lines
```

Comment:
what's the reason that you removed this comment?

Reply:
Probably got cut off while refactoring. I will re-add it.